hbase-resource-bundle/src/main/resources/supplemental-models.xml

```diff
@@ -332,6 +332,19 @@ under the License.
       </licenses>
     </project>
   </supplement>
+  <supplement>
+    <project>
+      <groupId>javax.servlet.jsp</groupId>
+      <artifactId>jsp-api</artifactId>
```

Comment:
weren't we already packaging this in the assembly? are we getting more than one of them from different groups or something?

Reply:
There is a jsp-api and javax.servlet.jsp-api artifact in javax.servlet.jsp package. Luckily both of them are on our classpath.

Reply:
Shouldn't we be able to exclude one of them then?

Reply:
excluded